### PR TITLE
Totalsegmentator updated loader

### PR DESCRIPTION
**Pull request by shevtsovalexey**

### CODE REVIEW

File: amid/totalsegmentator/dataset.py [hunk 1]
```diff
@@ -101,16 +96,19 @@ def affine(i, _base):
...
-        with unpack(_base, file) as (unpacked, is_unpacked):
+        with unpack(_base, file, ARCHIVE_ROOT, '.zip') as (unpacked, is_unpacked):
             if is_unpacked:
                 return nibabel.load(unpacked).affine
             else:
                 with open_nii_gz_file(unpacked) as image:
                     return image.affine
 
+    def spacing(i, affine: Output):
+        return get_spacing_from_affine(affine)
```
Comment (maxme1) [inline amid/totalsegmentator/dataset.py:107]: Maybe we should leave this to a transform? E.g. one that splits `affine` into `spacing`, `orientation` and `origin`

Comment (shevtsovalexey) [reply amid/totalsegmentator/dataset.py:110]: got it, but don't know where I can place this transform

Comment (maxme1) [reply amid/totalsegmentator/dataset.py:110]: `amid.nifty`?

Comment (shevtsovalexey) [reply amid/totalsegmentator/dataset.py:110]: idk; maybe amid.transforms?

Comment (maxme1) [reply amid/totalsegmentator/dataset.py:110]: yeah, ok

File: amid/transforms.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    def orientation(affine):
+        """Constructs an orientation matrix from the given affine matrix."""
+        return affine[:3, :3]
+
+    def origin(affine):
+        """Constructs an origin tensor from the given affine matrix."""
+        return affine[:-1, -1]
+
+    def voxel_spacing(orientation: Output):
```
Comment (maxme1) [inline amid/transforms.py:28]: maybe just `spacing`?

File: amid/transforms.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    >>> dataset = dataset >> ParseAffineMatrix()
+    >>> dataset.voxel_spacing(id_)
+    # array([1.5, 1.5, 1.5])
+    """
+
+    __inherit__ = True
+
+    def orientation(affine):
+        """Constructs an orientation matrix from the given affine matrix."""
+        return affine[:3, :3]
```
Comment (maxme1) [inline amid/transforms.py:22]: should divide it by spacing

**Review by maxme1 — APPROVED** [review]

**Pull request merged**